### PR TITLE
Method to retrieve raw JavaVM pointer

### PR DESCRIPTION
**Pull request by paulrouget**

### CODE REVIEW

File: src/wrapper/java_vm/vm.rs [hunk 1]
```diff
@@ -44,6 +44,11 @@ impl JavaVM {
         Ok(JavaVM(ptr))
     }
 
+    /// Returns underlying `sys::JavaVM` interface.
+    pub fn get_native_interface(&self) -> *mut sys::JavaVM {
```
Comment (dmitry-timofeev) [inline src/wrapper/java_vm/vm.rs]: I think 'native interface' might be confused with the JNI interface pointer (JNIEnv): https://docs.oracle.com/javase/10/docs/specs/jni/design.html#jni-interface-functions-and-pointers

I wonder if anyone can come up with a better name. I can think of `get_raw_vm_pointer` or `get_java_vm_pointer`?

File: src/wrapper/java_vm/vm.rs [hunk 2]
```diff
@@ -44,6 +44,11 @@ impl JavaVM {
...
+        self.0
```
Comment (dmitry-timofeev) [inline src/wrapper/java_vm/vm.rs]: I wonder if it will benefit from integration tests (e.g., a roundtrip test), or is it too simple to break :-) ?

Comment (paulrouget) [reply src/wrapper/java_vm/vm.rs]: Like, call `get_java_vm_pointer` follow by a `JavaVM::from_raw()`? And check that I can, for example, get back the env?

**Comment by paulrouget** [comment]
I've renamed the function. I'm not sure how to create a useful test though.

File: src/wrapper/java_vm/vm.rs (hunk 2 above)
Comment (dmitry-timofeev) [reply src/wrapper/java_vm/vm.rs]: Yes, I thought of something like that, but a test might get too complicated for such a simple getter.

**Comment by paulrouget** [comment]
Anything else needed to get this to land and be published?

**Comment by paulrouget** [comment]
@jechase ping

**Comment by paulrouget** [comment]
ping?

**Comment by paulrouget** [comment]
@dmitry-timofeev can this be published?

**Comment by dmitry-timofeev** [comment]
I believe we'll publish the next week (I don't have the permissions on crates).

**Pull request merged**